app/models/organization.rb

```diff
@@ -39,4 +40,20 @@ def next_reg_number_seq
     reg_number_seq
   end
 
+  def web_url
+    URI.parse(web)
+    rescue
+      false
+  end
+
+  def ensure_url
+    if web_url
+      return if web.blank?
```

Comment:
```ruby
def ensure_url
  return if web.blank?
    if web_url
```
Si no, la organización no es válida (podrías añadir `expect(org.web).to be_valid` en todos los ejemplos en los que supones que se debe aceptar (nil, blanco) y verificarlo)